<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duplicate Checker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_17">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Choose Folder : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="folderEdit"/>
      <widget class="QToolButton" name="folderButton">
       <property name="maximumSize">
        <size>
         <width>106</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+O</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openButton">
       <property name="maximumSize">
        <size>
         <width>106</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exportButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QWidget" name="visualizationView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Analyzing Files: Tracking Progress</string>
          </property>
         </widget>
         <widget class="QProgressBar" name="progressBar_1">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Analyzing hashes on 1k :</string>
          </property>
         </widget>
         <widget class="QProgressBar" name="progressBar_2">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Analyzing full hashes : </string>
          </property>
         </widget>
         <widget class="QProgressBar" name="progressBar_3">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="maximumSize">
        <size>
         <width>109</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Full Hash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hash on 1k</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTableView" name="duplicatesView_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableView" name="allFilesView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashDotDotLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>205</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Overall Duplicate File Size :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>205</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Excess Duplicate File Size :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>205</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Total Duplicate Files :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>205</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Total Unique Files : </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="total_size">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_duplicate_size">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_duplicate_files">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_unique_files">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableView" name="duplicatesView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>222</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Total Size for Chosen Duplicate : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>255</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Selected Excess Duplicate File Size :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>240</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Selected Duplicate Files Quantity :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Small Hash Duplicates :</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="total_size_single">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_duplicate_size_single">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_files_single">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="small_hash_count">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
